docs/2.19.x/how-to-guides/work-pools/kubernetes.mdx

```diff
@@ -336,6 +338,16 @@ worker:
 
 ```
 
+#### Using Prefect Server
+```
+worker:
+  serverApiConfig:
+  apiUrl: <dns or ip address of your Prefect Server instance>
```

Comment:
i think this should be invented one

Reply:
updated in [indent](https://github.com/PrefectHQ/prefect/pull/14040/commits/92b4daf35c4c48870df122ccb5f7196302c021a8)